INFO - 2024-01-23 00:04:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-23 00:05:05 --> mysqli_sql_exception: Table 'x_desa.user' doesn't exist in C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\ci4\e-smades\app\Models\Cms\UserModel.php(43): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\ci4\e-smades\app\Controllers\Cms\AuthController.php(41): App\Models\Cms\UserModel->getUserByEmail('admin@gmail.com')
#8 C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\CodeIgniter.php(941): App\Controllers\Cms\AuthController->signInProcess()
#9 C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cms\AuthController))
#10 C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\e-smades\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\e-smades\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-01-23 00:05:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'x_desa.user' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\Cms\UserModel.php(43): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\Cms\AuthController.php(41): App\Models\Cms\UserModel->getUserByEmail('admin@gmail.com')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cms\AuthController->signInProcess()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cms\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\e-smades\\public\\index.php')
CRITICAL - 2024-01-23 00:05:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'x_desa.user' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user`
WHERE `email` = \'admin@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user`
WHERE `email` = \'admin@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\Cms\UserModel.php(43): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\Cms\AuthController.php(41): App\Models\Cms\UserModel->getUserByEmail('admin@gmail.com')
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cms\AuthController->signInProcess()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cms\AuthController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\e-smades\\public\\index.php')
CRITICAL - 2024-01-23 00:05:05 --> [Caused by] mysqli_sql_exception: Table 'x_desa.user' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `user`
WHERE `email` = \'admin@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user`
WHERE `email` = \'admin@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user`
WHERE `email` = \'admin@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\Cms\UserModel.php(43): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\Cms\AuthController.php(41): App\Models\Cms\UserModel->getUserByEmail('admin@gmail.com')
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cms\AuthController->signInProcess()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cms\AuthController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\e-smades\\public\\index.php')
INFO - 2024-01-23 00:05:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-23 00:18:48 --> Error: Call to undefined function getListModule()
in APPPATH\Views\cms\templates\menu.php on line 9.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('cms/templates/menu', [], true)
 4 APPPATH\Views\public\templates\main.php(143): view('cms/templates/menu')
 5 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\ci4\\e-smades\\app\\Views\\public\\templates\\main.php')
 6 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 7 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('public/templates/main', [], true)
 8 APPPATH\Controllers\BaseController.php(62): view('public/templates/main', [...])
 9 APPPATH\Controllers\Home.php(21): App\Controllers\BaseController->render([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->index()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\e-smades\\public\\index.php')
CRITICAL - 2024-01-23 00:20:37 --> Error: Call to undefined function getListModule()
in APPPATH\Views\cms\templates\menu.php on line 9.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('cms/templates/menu', [], true)
 4 APPPATH\Views\public\templates\main.php(143): view('cms/templates/menu')
 5 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\ci4\\e-smades\\app\\Views\\public\\templates\\main.php')
 6 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 7 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('public/templates/main', [], true)
 8 APPPATH\Controllers\BaseController.php(62): view('public/templates/main', [...])
 9 APPPATH\Controllers\Home.php(21): App\Controllers\BaseController->render([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->index()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\e-smades\\public\\index.php')
INFO - 2024-01-23 14:04:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-23 14:05:35 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:185
Stack trace:
#0 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'x_desa', 3306, '', 0)
#1 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/app/Models/Cms/UserModel.php(43): CodeIgniter\BaseModel->first()
#7 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/app/Controllers/Cms/AuthController.php(41): App\Models\Cms\UserModel->getUserByEmail('admin@gmail.com')
#8 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(941): App\Controllers\Cms\AuthController->signInProcess()
#9 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cms\AuthController))
#10 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/dcdrcpia/e-smades.dcdrcpiak.id/index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/app/Models/Cms/UserModel.php(43): CodeIgniter\BaseModel->first()
#6 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/app/Controllers/Cms/AuthController.php(41): App\Models\Cms\UserModel->getUserByEmail('admin@gmail.com')
#7 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(941): App\Controllers\Cms\AuthController->signInProcess()
#8 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cms\AuthController))
#9 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/dcdrcpia/e-smades.dcdrcpiak.id/index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-23 14:05:35 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `m_user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH/Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH/Models/Cms/UserModel.php(43): CodeIgniter\BaseModel->first()
 6 APPPATH/Controllers/Cms/AuthController.php(41): App\Models\Cms\UserModel->getUserByEmail('admin@gmail.com')
 7 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\Cms\AuthController->signInProcess()
 8 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cms\AuthController))
 9 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-23 14:17:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-23 14:17:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-23 14:17:35 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'dcdrcpia_isyanto'@'localhost' (using password: YES) in /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:185
Stack trace:
#0 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(185): mysqli->real_connect('localhost', 'dcdrcpia_isyant...', 'asdfASDF1234', 'dcdrcpia_esmade...', 3306, '', 0)
#1 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/app/Models/Cms/UserModel.php(43): CodeIgniter\BaseModel->first()
#7 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/app/Controllers/Cms/AuthController.php(41): App\Models\Cms\UserModel->getUserByEmail('admin@gmail.com')
#8 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(941): App\Controllers\Cms\AuthController->signInProcess()
#9 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cms\AuthController))
#10 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/dcdrcpia/e-smades.dcdrcpiak.id/index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/app/Models/Cms/UserModel.php(43): CodeIgniter\BaseModel->first()
#6 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/app/Controllers/Cms/AuthController.php(41): App\Models\Cms\UserModel->getUserByEmail('admin@gmail.com')
#7 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(941): App\Controllers\Cms\AuthController->signInProcess()
#8 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cms\AuthController))
#9 /home/dcdrcpia/e-smades.dcdrcpiak.id/src/vendor/codeigniter4/framework/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/dcdrcpia/e-smades.dcdrcpiak.id/index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-23 14:17:35 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `m_user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH/Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH/Models/Cms/UserModel.php(43): CodeIgniter\BaseModel->first()
 6 APPPATH/Controllers/Cms/AuthController.php(41): App\Models\Cms\UserModel->getUserByEmail('admin@gmail.com')
 7 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\Cms\AuthController->signInProcess()
 8 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cms\AuthController))
 9 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-23 14:17:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-23 14:18:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-23 14:18:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
